A1: [0, 0, 0]
A2: [0, 0, 0]
A3: [0, 0, 0]
A4: [0, 0, 0]
A5: [0, 0, 0]
A6: [0, 0, 0]
A7: [0, 0, 0]
A8: [0, 0, 0]
B1: [0, 0, 0]
B2: [0, 0, 0]
B3: [0, 0, 0]
B4: [0, 0, 0]
B5: [0, 0, 0]
B6: [0, 0, 0]
B7: [0, 0, 0]
B8: [0, 0, 0]
C1: [0, 0, 0]
C2: [0, 0, 0]
C3: [0, 0, 0]
C4: [0, 0, 0]
C5: [0, 0, 0]
C6: [0, 0, 0]
C7: [0, 0, 0]
C8: [0, 0, 0]
D1: [0, 0, 0]
D2: [0, 0, 0]
D3: [0, 0, 0]
D4: [0, 0, 0]
D5: [0, 0, 0]
D6: [0, 0, 0]
D7: [0, 0, 0]
D8: [0, 0, 0]
E1: [0, 0, 0]
E2: [0, 0, 0]
E3: [0, 0, 0]
E4: [0, 0, 0]
E5: [0, 0, 0]
E6: [0, 0, 0]
E7: [0, 0, 0]
E8: [0, 0, 0]
F1: [0, 0, 0]
F2: [0, 0, 0]
F3: [0, 0, 0]
F4: [0, 0, 0]
F5: [0, 0, 0]
F6: [0, 0, 0]
F7: [0, 0, 0]
F8: [0, 0, 0]
G1: [0, 0, 0]
G2: [0, 0, 0]
G3: [0, 0, 0]
G4: [0, 0, 0]
G5: [0, 0, 0]
G6: [0, 0, 0]
G7: [0, 0, 0]
G8: [0, 0, 0]
GRIPPER_ORIENT: [0, 0.707, 0, 0.707]
GRIPPER_ORIENT_REST: [-0.03, 0.371, 0.02, 0.928]
H1: [0, 0, 0]
H2: [0, 0, 0]
H3: [0, 0, 0]
H4: [0, 0, 0]
H5: [0, 0, 0]
H6: [0, 0, 0]
H7: [0, 0, 0]
H8: [0, 0, 0]
OFFSET: 0.23
aruco: false
clear_octomap: false
color: black
escape_strategy: nearby
full_setup: true
goal_position_tolerance: 0.001
look_for: 5
max_acceleration_scaling_factor: 1.0
max_velocity_scaling_factor: 0.2
num_planning_attempts: 5
open_wide_right: true
planning_time: 5
simulation: true
time_limit: 3
